fvm/evm/handler/handler_test.go

```diff
@@ -506,6 +505,33 @@ func TestHandler_BridgedAccount(t *testing.T) {
 		})
 	})
 
+	t.Run("test call to cadence arch", func(t *testing.T) {
+		t.Parallel()
+
+		testutils.RunWithTestBackend(t, func(backend *testutils.TestBackend) {
+			blockHeight := uint64(123)
+			backend.GetCurrentBlockHeightFunc = func() (uint64, error) {
+				return blockHeight, nil
+			}
+			testutils.RunWithTestFlowEVMRootAddress(t, backend, func(rootAddr flow.Address) {
+				h := SetupHandler(t, backend, rootAddr)
+
+				foa := h.AccountByAddress(h.AllocateAddress(), true)
+				require.NotNil(t, foa)
+
+				orgBalance, err := types.NewBalanceFromAttoFlow(new(big.Int).Mul(big.NewInt(1e18), big.NewInt(10000)))
```

Comment:
why not instead:
```
types.Balance(10000)
```

Reply:
that won't work because of the why Cadence stores Flow, it requires some adjustment , i added a utility function at [a6abd9b](https://github.com/onflow/flow-go/pull/5233/commits/a6abd9b9ae7efe8c96f8b3ae11a11004bfefa3c7)